ivadomed/utils.py

```diff
@@ -619,6 +619,27 @@ def threshold_predictions(predictions, thr=0.5):
     return thresholded_preds
 
 
+def segment_volume(tensor_data, model_name):
```

Comment:
Function to segment a `torch` tensor. Minimal complexity, so that we can build on it.

We assume the following process:
1. Call the `ivadomed` loader from the CLI
2. Stack the slices (`torch` tensor format)
3. Call the `ivadomed.utils.segment_volume()` function
4. Undo the transforms on the segmented slices within the CLI

Reply:
We decided during the last SCT meeting that `segment_volume` should receive the image filename as input and deal with the loading in it.